Source/WebCore/page/Quirks.cpp

```diff
@@ -703,6 +703,21 @@ bool Quirks::shouldAvoidScrollingWhenFocusedContentIsVisible() const
     return needsQuirks() && m_quirksData.shouldAvoidScrollingWhenFocusedContentIsVisibleQuirk;
 }
 
+// Some input only specify image/* as an acceptable type, which is failing sometimes for certains domain names
+// which do not support HEIC.
+bool Quirks::shouldTranscodeHeicImagesForURL(const URL& url)
+{
+    if (!needsQuirks())
+        return false;
+
+    auto host = url.host();
+    // zillow.com rdar://79872092
+    if (host == "zillow.com"_s)
+        return true;
```

Comment:
This is probably fine for `Zillow.com`, which is a simple URL. If we wanted to check more complicated things we would want to use:
` auto quirksDomainString = RegistrableDomain(quirksURL).string();`